citi-onboarding/Aqua-Jr
# Feature/about us

## What I did
- Made the About Us section.
## How to test
- On one terminal you should type the following commands to access the client folder and run yarn start
```      
    cd client
    yarn start
```  
- If the previous step was followed correctly you should now have a blank localhost300 page open on your server
- Moving on, go to your second terminal and access the server folder, where you should run either node index.js or nodemon index.js
```  
    cd server
    nodemon index.js
```  
- Now you're all set and should be able to view the website on your localhost3000 page. To properly test it, put it through the following test: 
  - Change your screen size both randomly and following the sizes suggested by the server and check if it looks just like the mockup
